javalib/src/main/scala/java/io/DataInputStream.scala

```diff
@@ -124,7 +124,7 @@ class DataInputStream(in: InputStream)
     var v = in.read()
     if (v == -1) null
     else {
-      val builder = new StringBuilder
+      val builder = new java.lang.StringBuilder
```

Comment:
I always just alias `java.lang` -> `jl` and `java.util` -> `ju`. I think that is pretty nice unless it is adding extra symbols or something.